service/src/test/java/uk/nhs/adaptors/gpc/consumer/FindAPatientDocsTest.java

```diff
@@ -0,0 +1,24 @@
+package uk.nhs.adaptors.gpc.consumer;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+import uk.nhs.adaptors.gpc.consumer.filters.SearchForDocumentsGatewayFilterFactory;
+import uk.nhs.adaptors.gpc.consumer.utils.FindAPatientDocsUtil;
+
+public class FindAPatientDocsTest {
+    private static final String GPC_BASE_URL = "http://localhost:8110";
+    private static final String GPC_CONSUMER_BASE_URL = "http://localhost:8080";
+    private static final String RESPONSE_BODY_TEMPLATE = "{\"url\": \"%s/example\"}";
+    private static final String GPC_RESPONSE_BODY = String.format(RESPONSE_BODY_TEMPLATE, GPC_BASE_URL);
+
+    @Test
+    public void When_GPCBaseUrlPresentInBody_Expect_GPCConsumerBaseUrlReplacementInBody() {
```

Comment:
Can you add some additional tests to complete the coverage?
1) body contains more than one url
2) body contains no urls
3) body is empty

Would it be useful to grab an actual "Search for a patient's documents" response and use that in the test?